Dockerfile

```diff
@@ -1,7 +1,19 @@
-FROM alpine:3.10
+# This is a two stage build
+# first we need build-base and friends to build our custom wren_cli
+FROM alpine:3.13
+RUN apk add --no-cache jq coreutils git build-base bash
```

Comment:
Do you need `jq` and/or `bash` for this build stage?

Reply:
Nope, i've since removed.